src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -572,13 +574,18 @@ public MetadataEnumeration<SnapshotSummary> retrieveSnapshots(
             + " datasetIds: "
             + StringUtils.join(datasetIds, ",")
             + " tags: "
-            + StringUtils.join(tags, ","));
+            + StringUtils.join(tags, ",")
+            + " duosIds: "
+            + StringUtils.join(duosIds, ","));
```

Comment:
No need to use `StringUtils` when you have a `List<String`
```suggestion
            + String.join(",", tags)
            + " duosIds: "
            + String.join(",", duosIds));
```

Also I think the recommended form is to use `{}` instead of string concatenation; also unless we regularly run with `DEBUG` logging this is doing work that's unused most of the time and could be wrapped in `if (logger.debugEnabled()) { ... }`

Reply:
new intellij for the win on this one!